### ISSUE TITLE
is it possible to know if a tweet is original or if it's a retweet?

### ISSUE THREAD

**Issue by Nestor75** [haccer/twint#94]
### Initial Check 
Make sure you've checked the following.

- [x] Python version is 3.5 or higher.
- [x] Using the latest version of Twint.

If this is a feature request please specify in the title:

Example:
> [REQUEST] More features!

### Command
Please provide the exact command ran including the username/search so I may reproduce the issue.

### Description of Issue
It's a doubt rather than a issue, I can get the list of tweets matching with a criteria but, Is there a way to know is a tweet is original of if it is a retweet from other one? 

and if it is possible, is it possible to know who wrote the original tweet?
many thanks 
### OS Details
Using Windows, Linux? Linux  What OS version? Ubuntu 16

**Comment by haccer** [comment]
Hi @Nestor75,

Could you elaborate a bit?

**Comment by Nestor75** [comment]
Yes of course, 

Let's give an explample:

- Original_user tweets  a tweet with id "00000000001" and the text "this is an example #example"  

- user A retweets the tweet with id "00000000001" and the text "this is an example #example" from Original user-> this tweet would have anothee different ID = "00000000002"

- user B retweets the tweet with id "00000000002" and the text "this is an example #example" from user A-> this tweet would have anothee different ID = "00000000003"

and all of them are retweeted by many other users

imagine I search by the hashtag "#example" and I got all the above tweets plus all the tweets from other users. 

Is there a way to find out which was the original tweet and therefore the original user?

**Comment by haccer** [comment]
> tl;dr they're all original

I'm still a little confused by the question, but I don't think a Tweet is assigned a new tweet-id when it's retweeted... plus RT'd tweets aren't visible from Twitter's search, only the Tweets themselves.

I do have a method to collect Retweets, but I haven't implemented it into the project yet.

Hope this answers your question.

**Comment by DavidPerea** [comment]
Therefore, all tweets, which you get with "twint", from a user are your own tweets. True?

So do not we get the tweets that are retweets? That is, not the own tweets of the user if not the tweets that the user have retweeted of other users.

The truth that it would be very useful to know the tweets that have retweeted a user to identify their activity and interest.

Thank!!!!

**Comment by hudmb** [comment]
You can use Twitter's own parameter:-

filter:retweets

If you only want to fetch RTs, but it would also be useful to have a column in the output that identified whether the tweet was organic, a reply, or a retweet.

**Comment by DavidPerea** [comment]
Sorry @hudmb , I didn't understand what you mean with the filter. 

Could you tell me what command I should use to get the tweets that a user has retweeted? 

Thank you very much!!!

**Comment by hudmb** [comment]
In a search parameter you can use something like this: -

-s "from:realdonaldtrump filter:retweets"

This should retrieve all RTs by Donald Trump, as far as I know.

Maybe @haccer could confirm if the filter:retweets could be used with the user command as well as the search command?

**Comment by hpiedcoq** [comment]
not possible according to me.

**Comment by haccer** [comment]
Hi @hudmb ,

`filter:retweets` doesn't work with Twitter anymore, retweeted Tweets aren't visible from Twitter's search
